test/unit/lib/is-array.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Do we care that we're not explicitly testing the logic of the toString version? I think it's fine, but wanted to check.